rdmo/core/settings.py

```diff
@@ -332,7 +334,8 @@
 
 PROJECT_SEND_INVITE = True
 
-PROJECT_REMOVE_VIEWS = True
+PROJECT_VIEWS_SYNC = True
+PROJECT_TASKS_SYNC = True
```

Comment:
should these actually be enabled by default?

Reply:
I think not.